src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -352,6 +355,17 @@ private VariableDesignationSyntax Transform( VariableDesignationSyntax des )
 			_ => UnhandledSyntax( des )
 		};
 
+	private SeparatedSyntaxList<CollectionElementSyntax> Transform(
+		SeparatedSyntaxList<CollectionElementSyntax> original
+	) => SyntaxFactory.SeparatedList(
+		original.Select( elem => elem switch {
+			ExpressionElementSyntax ee => ee.WithExpression( Transform( ee.Expression ) ),
+			SpreadElementSyntax sp => sp.WithExpression( Transform( sp.Expression ) ),
+			_ => UnhandledSyntax( elem )
+		} ),

```

Comment:
Particular reason this isn't just `original.Select( Transform )` or something, with the top-level Dispatch handling Spread?